package.json

```diff
@@ -6,23 +6,26 @@
   "scripts": {
     "frontend": "nx serve frontend",
     "deploy": "nx deploy frontend",
-    "buildall": "nx run-many --target=build"
+    "buildall": "nx run-many --target=build",
+    "mini-ci": "nx affected -t test,lint --max-warnings=0"
   },
   "devDependencies": {
     "@babel/preset-react": "^7.18.6",
     "@emotion/babel-plugin": "11.11.0",
     "@nx/cypress": "16.4.3",
     "@nx/eslint-plugin": "16.4.3",
-    "@nx/jest": "16.4.3",
-    "@nx/js": "16.4.3",
+    "@nx/jest": "16.5.1",
```

Comment:
im not sure about bumping nx module version without bumping `nx`. i would recommend running `nx migrate latest` to keep them in sync.

Reply:
That wasn't the intension, `yarn` seems to bump it automatically when I add `@nx/plugin`